configure.ac

```diff
@@ -1444,13 +1444,22 @@ if test -n "$LIBAIO"; then
 fi
 
 BUILD_LIBURING=no
-AC_CHECK_HEADERS([liburing.h])
-AC_CHECK_LIB([uring],[io_uring_queue_init],[LIBURING="-luring"])
+case $host_os in
+    linux*)
+        AC_ARG_ENABLE([linux-io_uring],
+        AC_HELP_STRING([--disable-linux-io_uring],
+                             [Disable io-uring in POSIX xlator.]))
+        if test "x$enable_linux_io_uring" != "xno"; then
+            AC_CHECK_HEADERS([liburing.h],
+            [AC_DEFINE(HAVE_LIBURING, 1, [io-uring based POSIX enabled])
```

Comment:
Idention looks a bit weird?

Reply:
I placed line 1453 onwards at 4 spaces from the 'if' column. Not sure  what you mean...